src/Tests/ClientConcepts/LowLevel/DirectStreaming.cs

```diff
@@ -79,5 +74,39 @@ public void EnableDirectStreamingOnSuccess()
 			response = client.SearchAsync<object>(s => s).Result;
 			assert(response);
 		}
+
+		[U]
+		public void DisableDirectStreamingOnRequest()
+		{
+			Action<IResponse> assert = r =>
+			{
+				r.ApiCall.Should().NotBeNull();
+				r.ApiCall.RequestBodyInBytes.Should().NotBeNull();
+				r.ApiCall.ResponseBodyInBytes.Should().NotBeNull();
+			};
+
+			var client = TestClient.GetFixedReturnClient(new { });
+			var response = client.Search<object>(s => s.RequestConfiguration(r => r.DisableDirectStreaming()));
+			assert(response);
+			response = client.SearchAsync<object>(s => s.RequestConfiguration(r => r.DisableDirectStreaming())).Result;
+			assert(response);
+		}
+
+		[U]
+		public void EnableDirectStreamingOnRequest()
+		{
+			Action<IResponse> assert = r =>
+			{
+				r.ApiCall.Should().NotBeNull();
+				r.ApiCall.RequestBodyInBytes.Should().BeNull();
+				r.ApiCall.ResponseBodyInBytes.Should().BeNull();
+			};
+
+			var client = TestClient.GetFixedReturnClient(new { }, 200, c => c.DisableDirectStreaming());
+			var response = client.Search<object>(s => s.RequestConfiguration(r => r.DisableDirectStreaming(false)));
+			assert(response);
+			response = client.SearchAsync<object>(s => s.RequestConfiguration(r => r.DisableDirectStreaming(false))).Result;
+			assert(response);
+		}
```

Comment:
My bad, I missed where you set DisableDirectStreaming globally in the second test.  LGTM 👍